rust/streaming_pipe_in.rs

```diff
@@ -0,0 +1,19 @@
+//Script reads text from a named pipe and writes it to stdout, capitalized
+
+use std::env;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader};
```

Comment:
```suggestion
use std::io::{BufRead, BufReader};
```

It's unneeded. The `BufRead` is needed, however... it's getting used implicitly somewhere.